### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by szalik-rg351** [dhwz/es-theme-351elec-carbon#1]
are you doing this for particular system? We can work togheter if for elec :)

**Comment by dhwz** [comment]
Yeah, I'm doing it for my new RG351M (currently with 351ELEC) which I just got this week.
There were some things which didn't look good so I tried to change them in first place.
Sure if you agree with my changes I can do a pull request when I'm finished.

**Comment by szalik-rg351** [comment]
Yeah you done many nice catches here, very detailed work. Specially on Sega. I notice they were abit off, but i just added them as a placeholder. Im happy with most of your changes so it will be great if you do a pull request, so it will be applied for main theme on packages for others :)

**Comment by szalik-rg351** [comment]
btw if you want to add system for the game what you don't have any games for. You can simply create new.txt and change extension for the system needs : ) it will help you get it displayed. extensions you can check on es_config or into wiki

**Comment by dhwz** [comment]
Yes, I'll improve the other systems too. I'll continue next weekend :)

**Comment by szalik-rg351** [comment]
im glad you working on this :) i will do small updates after you finish too, but i will make sure to respect your changes : ) just some backgrounds i find in better quality, or slighlty better ones 🗡️
